paasta_tools/cli/cmds/logs.py

```diff
@@ -461,18 +471,22 @@ def prettify_log_line(
                 }
             )
         else:
+            log_line = {
+                "timestamp": prettify_timestamp(parsed_line["timestamp"]),
+                "component": prettify_component(parsed_line["component"]),
+                "cluster": " [%s]" % parsed_line["cluster"],
```

Comment:
nit: `"" if strip_cluster else f" [{parsed_line['cluster']}]"`? same for instance

(just inlining the if-else, and using f-strings, which is more py3)

Reply:
Done